src/Native/System.Native/pal_io.cpp

```diff
@@ -337,43 +316,39 @@ int32_t ReadDirR(DIR* dir, void* buffer, int32_t bufferSize, DirectoryEntry* out
     return 0;
 }
 
-extern "C" 
-DIR* OpenDir(const char* path)
+extern "C" DIR* OpenDir(const char* path)
 {
     return opendir(path);
 }
 
-extern "C" 
-int32_t CloseDir(DIR* dir)
+extern "C" int32_t CloseDir(DIR* dir)
 {
     return closedir(dir);
 }
 
-extern "C"
-int32_t Pipe(int32_t pipeFds[2], int32_t flags)
+extern "C" int32_t Pipe(int32_t pipeFds[2], int32_t flags)
 {
     switch (flags)
     {
-    case 0:
-        break;
-    case PAL_O_CLOEXEC:
-        flags = O_CLOEXEC;
-        break;
-    default:
-        assert(false && "Unknown flag.");
-        errno = EINVAL;
-        return -1;
+        case 0:
+            break;
+        case PAL_O_CLOEXEC:
+            flags = O_CLOEXEC;
+            break;
+        default:
+            assert(false && "Unknown flag.");
+            errno = EINVAL;
+            return -1;
     }
 
 #if HAVE_PIPE2
     return pipe2(pipeFds, flags);
 #else
-    return pipe(pipeFds); // CLOEXEC intentionally ignored on platforms without pipe2.
+    return pipe(pipeFds);         // CLOEXEC intentionally ignored on platforms without pipe2.
```

Comment:
LLVM style sets the SpacesBeforeTrailingComments option at a value so all trailing comments will be preceded by X number of spaces